WebRoot/js/zimbraAdmin/common/ZaModel.js

```diff
@@ -437,6 +437,133 @@ ZaModel.getMailPollingIntervalChoices = function () {
 
 ZaModel.MAIL_POLLING_INTERVAL_CHOICES = ZaModel.getMailPollingIntervalChoices;
 
+ZaModel.getMailRecallChoices = function () {
+    return [
+        {
+            value: "1",
+            label: "1 " + ZaMsg.LBL_min
+        },
+        {
+            value: "2",
+            label: "2 " + ZaMsg.LBL_minute
+        },
+        {
+            value: "3",
+            label: "3 " + ZaMsg.LBL_minute
```

Comment:
can't we write a for loop to construct this object instead of writing like this?

Reply:
changes done